src/docs/truffle/reference/configuration.md

```diff
@@ -271,13 +272,36 @@ module.exports = {
           enabled: <boolean>,
           runs: <number>   // Optimize for how many times you intend to run the code
         },
-        evmVersion: <string> // Default: "petersburg"
+        evmVersion: <string> // Default: "istanbul"
+      },
+      modelCheckerSettings: {
+        // contains options for SMTChecker
       }
     }
   }
 }
 ```
-For more information, please see the Solidity documentation on [Compiler Input and Output JSON Description](http://solidity.readthedocs.io/en/develop/using-the-compiler.html#compiler-input-and-output-json-description).
+For more information, please see the Solidity documentation on [Compiler Input JSON Description](https://docs.soliditylang.org/en/latest/using-the-compiler.html#input-description) for the various possible settings.  Note that customizing `stopAfter` and `outputSelection` are not currently supported.
+
+### vyper
+
+Vyper compiler settings.  Similar to the `solc` settings, but not as extensive.  In particular, specifying a Vyper version is not currently supported; your local Vyper installation will always be used.
+
+Truffle config example:
```

Comment:
Should we note here that this is `truffle-config.js`? Maybe it is evident, I'm not sure.

Reply:
I think it's evident; we don't explicitly mention this on the documentation for any of the other settings.  We just mention it once at the top of the page.  This is the page for the config documentation, after all.